internal/api/core/describe.go

```diff
@@ -19,3 +31,45 @@ func SetAPIDescription(description APIDescription) {
 func GetAPIDescription() APIDescription {
 	return apiDescription
 }
+
+func GetDescriptionFromHandler(basePath string) (string, error) {
+	absPath := makeAbsLocalAPIPath(basePath) + ".go"
+	if !util.IsFile(absPath) {
+		return "", fmt.Errorf("Unable to find file path to API Handler. Endpoint: '%s'. Expected path: '%s'.", basePath, absPath)
+	}
+
+	fset := token.NewFileSet()
+	node, parseErr := parser.ParseFile(fset, absPath, nil, parser.ParseComments)
+	if parseErr != nil {
+		return "", fmt.Errorf("Error while parsing file to get API description '%v'. Endpoint: '%s'.", parseErr, basePath)
+	}
+
+	for _, decl := range node.Decls {
+		function, ok := decl.(*ast.FuncDecl)
+		if !ok {
+			continue
+		}
+
+		if function.Name == nil {
+			continue
+		}
+
+		if function.Doc == nil {
+			continue
+		}
+
+		if strings.Contains(function.Name.Name, "Handle") {
+			return strings.TrimSpace(function.Doc.Text()), nil
+		}
```

Comment:
When you move this to util, makes some test cases for non-existent and empty (whitespce) comments.